lib/rules/max-empty-lines/index.js

```diff
@@ -23,47 +21,28 @@ const rule = function (max) {
     if (!validOptions) {
       return
     }
-
+    
+    // `//`-comments are need to be processed by postcss-scss
+    // To do that we have to use .toString()
     const rootString = root.toString()
```

Comment:
Try to use `root.source.input.css`, because we can have potential incorrect position on non standard syntax.